api/__tests__/fixtures/get-jwt-payload-user.json

```diff
@@ -27,7 +22,7 @@
     "email_verified": true,
     "name": "Jane Doe",
     "preferred_username": "jane1100@idir",
-    "given_name": "Yuhan",
-    "family_name": "Freeman",
+    "given_name": "Jane",
+    "family_name": "Doe",
     "email": "jane@example.com"
 }
```

Comment:
```suggestion
}

```
Nit: ending newline

Reply:
Looks like the rule of ending newlines doesn't go on .json file easily, at least not within the VS code?